content/en/docs/ops/ambient/usage/ztunnel/index.md

```diff
@@ -0,0 +1,656 @@
+
+We note that the figure also shows traffic from pod C3 destined to destination pod S1 on worker node W2 also traverses the local ztunnel proxy instance so that L4 traffic management functions such as L4 Authorization and L4 Telemetry are enforced identically on traffic whether or not it crosses a node boundary. 
+
+#### Ztunnel datapath via waypoint
+The next figure illustrates the data path for a different use case where there is a need to traverse through an interim Waypoint proxy. This is the case when the application service requires advanced L7 traffic routing, management or policy handling. Here ztunnel L4 networking is used to tunnel traffic torwads a Waypoint proxy for L7 processing and then mapped via a second HBONE tunnel towards the ztunnel on the node hosting the selected service destination pod.  In general the Waypoint proxy may or may not be located on the same nodes as the source or destinaton pods. 
+
+{{< image width="100%"
+    link="ztunnel-waypoint-datapath.png"
+    caption="Ztunnel datapath via an interim waypoint"
+    >}}
+
+
+#### Ztunnel datapath hairpinning 
+Finally we illustrate the concept of hairpinning in the data plane via the next figure. We noted earlier that traffic is always sent to a destination pod by first sending it to the ztunnel proxy on the same node as the destination pod.  But what if the sender is either completely outside the Istio ambient mesh and hence does not initiate HBONE tunnels to the destination ztunnel first ? or what if the sender is a malicious entity trying to send traffic directly to an ambient pod destination bypassing the destination service proxy processing ?? 
+
+There are two scenarios here both of which are depicted in the following figure. In the first scenario, traffic stream B1 is being received by node W2 outside of any HBONE tunnel and addressed directly to ambient pod S1's IP address for some reason (maybe because the traffic source is not an ambient pod). As shown in the figure, the ztunnel traffic redirection logic will intercept such traffic and redirect it via the local ztunnel proxy for destination side proxy processing and possible filtering based on AuthorizationPolicy prior to sending it into pod S1. In the second scenario, traffic stream G1 is being received by the ztunnel proxy of node W2 (possibly over an HBONE tunnel) but the ztunnel proxy checks that the destination service requires waypoint processing and yet the source sending this traffic is not a waypoint or is not associated with this destination service. In this case. again the ztunnel proxy hairpins the traffic towards one of the waypoints associated with the destination service from where it can then be delivered to any pod implementing the destination service (possibly to pod S1 itself as shown in the figure). 
+ 
+
+{{< image width="100%"
+    link="ztunnel-hairpin.png"
+    caption="Ztunnel traffic hairpinning"
+    >}}
+
+### Note on HBONE
```

Comment:
HBONE section may make more sense as an introductory section. Feels a little unnatural here, after its been used quite a bit with very little explanation.

Reply:
Having this earlier was not necessary since all that mattered initially was it was some form of tunneling but otherwise was transparent to the user and didnt need any configuration. Once the user has understood some basics of the ztunnel proxy by the time they get to this point, it helps to add a bit more detail hence imo this location is probably reasonable.